src/client/feed/feedActions.js

```diff
@@ -43,13 +42,16 @@ export const getMoreFeedContent = ({ sortBy, category, limit = 20 }) => (
   getState,
   { steemAPI },
 ) => {
-  const { feed, posts } = getState();
-  const feedContent = getFeedContentFromState(sortBy, category, feed, posts.list);
+  const { feed, ...state } = getState();
```

Comment:
any reason for making a copy here? Couldn't it just be `const state = getState();`const { feed } = state;`?

Reply:
Right, I think we could even just use `state.feed`.